### DISCUSSION TITLE
Welcome to phoenix Discussions!

### DISCUSSION THREAD

**Discussion by abose** [phcode-dev/phoenix#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BigDoughieG** [comment]
Anyone else having problems with desktop phoenix 4.03? I am having a problem with selecting text and getting to recognise when I have clicked on code to insert more. MacoS 15.1.1 (24B91)

**Reply by abose** [reply]
Can you explain the exact issue and steps to reproduce it

**Comment by BigDoughieG** [comment]
Hi Arun,

Strangely I can’t seem to reproduce the problem now. I had gone back to using phoenix 3.10.0 having spent a whole day with the problem but now it seems to be working normally.

The main aspect of the problem was when I clicked into a page I was not getting an insertion point and had to have three or four goes clicking about the page until I got in. It was as if clicking was not putting focus on the page.

Sorry I am unable to reproduce it for now. I will continue to use 4.03 and if the problem reappears I will make a video and forward it to you.

Douglas

**Comment by tayyebi** [comment]
Just wanted to thank the team for the amazing Phoenix. Competitive features, such as web app. Nice job, well done. <3

**Reply by devvaannsh** [reply]
Thank you for your kind words! We're glad you're enjoying Phoenix. The next update will make Phoenix even more powerful with built-in Emmet support, a tab bar, and many more features. ⚡

Don't forget to spread the word about Phoenix Code! 🚀

**Reply by tayyebi** [reply]
It would be even better, if you decrease dependencies to the internet, including but no limited to different feeds, etc. Keeping it minimalistic will help you achieve a bigger share of the market. Let's Rock!
